docker/README.md

```diff
@@ -0,0 +1,38 @@
+# Containerizing Dell EMC OpenManage Ansible Modules
+Containerized image of Dell EMC OpenManage Ansible Modules can be built using the dockerfile available [here](./Dockerfile). We will be using Docker Engine for containerizing OpenManage Ansible Modules. More information about Docker Engine and install instructions can be found [here](https://docs.docker.com/engine/).
+
+## Build
+Download [Dockerfile](./Dockerfile) to the machine where Docker Engine is installed and run below command. 
+```bash
+docker build -t dellemc/openmanage-ansible-modules .
+```
+Upon success, new image with name _dellemc/openmanage-ansible-modules_ will be created. One can verify successful creation of image using command `docker image ls` which will list newly created image with repository name as _dellemc/openmanage-ansible-modules_.
+
+Note: 
+1. This command need to be run from same directory where Dockerfile is downloaded. If not use the option `--file` to specify the path to Dockerfile.
+1. Docker file shown here uses the Docker official image [python:3](https://hub.docker.com/_/python) as base. Change this value is you want to use other python images as base. 
+
+## Usage
+`dellemc/openmanage-ansible-modules` docker image contains no playbook or configuration. One need to pass playbook, var files and inventory for running a task
```

Comment:
Provide the overview on where administrator can find the OMAM playbook example and how to use it

Reply:
will add link to example playbooks